docs/en/compute/Spark.md

```diff
@@ -13,22 +13,22 @@ This guide describes how to configure [Apache Spark](http://spark-project.org/)
 
 ## Overview
 
-Applications of Spark 1.1 or later can access an Alluxio cluster through its
-HDFS-compatible interface out-of-the-box.
+Applications using Spark 1.1 or later can access Alluxio through its
+HDFS-compatible interface.
 Using Alluxio as the data access layer, Spark applications can transparently access
-data in many different types and instances of persistent
-storage services (e.g., AWS S3 buckets, Azure Object Store buckets, remote HDFS deployments
-and etc). Data can be actively fetched or transparently cached into Alluxio to speed up
-the I/O performance especially when Spark deployment is remote to data.
+data in many different types of persistent storage services (e.g., AWS S3 buckets,
+Azure Object Store buckets, remote HDFS deployments and etc).
+Data can be actively fetched or transparently cached into Alluxio to speed up
+I/O performance especially when the Spark deployment is remote to the data.
 In addition, Alluxio can help simplify the architecture by decoupling
-compute and physical storage. When the real data path in persistent under storage is
-hidden from Spark, a change to under storages can be independent from application logic; meanwhile
-as a near-compute cache Alluxio can still provide compute frameworks like Spark data-locality.
+compute and physical storage. When the data path in persistent under storage is
+hidden from Spark, changes to under storage can be independent from application logic;
+meanwhile, as a near-compute cache, Alluxio can still provide compute frameworks data-locality.
 
 ## Prerequisites
 
-* Setup Java for Java 8 Update 60 or higher (8u60+), 64-bit.
-* Alluxio has been set up and is running.
+* Java 8 Update 60 or higher (8u60+), 64-bit.
+* An Alluxio is set up and is running.
```

Comment:
nit: "An Alluxio" feels weird

Reply:
whoops - meant to be "An Alluxio cluster"